index.css

```diff
@@ -0,0 +1,44 @@
+body{
+    background-image: url("/img/jess-bailey-l3N9Q27zULw-unsplash.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.main-container{
+    text-align: center;
+}
+
+.main-heading{
+    font-size: 80px;
+    font-weight: bold;
+    color: orangered;
+}
+
+.main-info p{
+    font-size: 25px;
+}
+
+.btn-Explore{
```

Comment:
try to stay consistent with your naming conventions. (btn-explore)

Reply:
yes!! Thanku 
i just read about outline : none , I will avoid it from now!